### DISCUSSION TITLE
RGBGenie/Sunricher undocumented/hidden parameter for No. of channels

### DISCUSSION THREAD

**Discussion by jhein05** [zwave-js/zwave-js#3086] [General] [answered]
Hi,

I found that some of the Sunricher/RGBGenie LED controllers have an undocumented parameter (4) that allows you to change the controller type. The options are 1: monochrom (1 channel) 2: CCT (tunable white; 2 channels); 3: RGBW (4 channels).

This definitely works for mine RGBGenie ZW-1001 identified as Sunricher ZV9101
It is highly likely that is also works for RGBGenie ZW-1005 

I am not a developer, so while I can probably hack it into my configuration files, it would surely be not good enough for your standards. I guess I just leave this here :)

**Comment by AlCalzone** [comment, marked as answer]
> I am not a developer, so while I can probably hack it into my configuration files, it would surely be not good enough for your standards. I guess I just leave this here :)

Or you could try and we'll guide you through it ;) I'll turn this into an issue, so we can properly track it.

**Reply by AlCalzone** [reply]
==> https://github.com/zwave-js/node-zwave-js/issues/3089
